android/app/src/main/res/values/strings.xml

```diff
@@ -9,7 +9,9 @@
     <string name="copyAndPasteProtection_title">Copy&amp;Paste Protection</string>
     <string name="copyAndPasteProtection_description">Disable the ability to copy from or paste into any text inputs in the app.</string>
     <string name="serverUrl_title">Mattermost Server URL</string>
-    <string name="serverUrl_description">Set a default Mattermost server URL.</string>
+    <string name="serverUrl_description">Set a default Server URL for your Mattermost instance.</string>
+    <string name="serverUrl_title">Mattermost Server Name</string>
+    <string name="serverUrl_description">Set a default Server Name for your Mattermost instance.</string>
```

Comment:
This are EMM configuration values that will appear with the metadata (see TestDPC if you want to know more)